### PR TITLE
CMS-309 ie fixes

### PR DESCRIPTION
**Pull request by lory87**
**This pull request fixes the following issues**:

- https://divio-ch.atlassian.net/browse/CMS-309

Before:
screen shot 2015-09-14 at 14 41 47

After:
screen shot 2015-09-14 at 14 41 57

Before:
screen shot 2015-09-14 at 16 25 07

After:
screen shot 2015-09-14 at 16 23 24

Before:
screen shot 2015-09-14 at 16 28 51

After:
screen shot 2015-09-14 at 16 28 56

### CODE REVIEW

File: djangocms_admin_style/sass/components/_forms.scss [hunk 1]
```diff
@@ -146,7 +146,8 @@ form {
         appearance: none;
         font-size: 18px;
         font-weight: normal;
-        padding: 10px 40px 10px 20px;
+        min-width: 160px;
+        padding: 10px 10px 10px 20px;
```
Comment (dhorelik) [inline djangocms_admin_style/sass/components/_forms.scss]: `10px 10px 20px`

**Comment by dhorelik** [comment]
apart from the minor comment, lgtm

File: djangocms_admin_style/sass/components/_forms.scss (hunk 1 above)
Comment (lory87) [inline djangocms_admin_style/sass/components/_forms.scss]: hey @dhorelik it's not the same I want to have left 20px not on bottom.

Comment (dhorelik) [inline djangocms_admin_style/sass/components/_forms.scss]: ouch, my bad, sorry

Comment (lory87) [inline djangocms_admin_style/sass/components/_forms.scss]: no worries ;) :+1:

**Pull request merged**